pkg/services/alerting/notifiers/pagerduty.go

```diff
@@ -100,6 +96,23 @@ func (pn *PagerdutyNotifier) Notify(evalContext *alerting.EvalContext) error {
 
 	payloadJSON := simplejson.New()
 
+	// set default, override in following case switch if defined
+	payloadJSON.Set("component", "Grafana")
+
+	for _, tag := range evalContext.Rule.AlertRuleTags {
+		customData.Set(tag.Key, tag.Value)
```

Comment:
Do you want to set the same field both in customData and payloadJson if a tag is in the PD API? Asking because I'm unsure.

Reply:
@aknuds1  

I think so yes... 

IIRC customData shows up in the actual pagerduty event which is passes to thee client, like mobile..  

<img width="290" alt="image" src="https://user-images.githubusercontent.com/58234/72008366-a57d8e00-3221-11ea-8977-65633676a459.png">

Similar to thresholds

However adding them to the payload portion allows you to classify and filter your events using Pagerduty's tooling via the UI.  You can see in the v2 api `class`, `group` and `component` have special meaning.  However I also think seeing them in the client, ie mobile would be helpful, so i put them in both.. hope that makes sense

Reply:
@aknuds1 / @papagian  - With regards to jsonassert, I wasn't sure of another way to test the json results of those methods... I just went w/ the tool I knew.. If you feel strongly about not adding this dependency I certainly understand... I just hope you may have some input into a more idiomatic way to accomplish what I am after here :)

Reply:
we good on this one ?

Reply:
Yeah I'll resolve it.